spec/display_spec.rb

```diff
@@ -28,20 +28,30 @@
       end.to output('Choose a position from 1-9: ').to_stdout
     end
 
-    it 'returns the users move as an integer' do
+    it 'returns the users move' do
       allow($stdin).to receive(:gets).and_return('1')
       move = display.ask_for_move
-      expect(move).to eq(1)
+      expect(move).to eq('1')
     end
   end
 
   context 'Displaying the current player:' do
     display = Display.new
-    
+
     it 'displays "The current player is x"' do
       expect do
         display.show_current_player('x')
       end.to output("The current player is x\n").to_stdout
     end
   end
+
+  context 'Displays message when user enters incorrect move' do
+    display = Display.new
+
+    it 'displays "22 is an invalid move."' do
+      expect do
+        display.invalid_move_message('22')
+      end.to output("22 is an invalid move.\n").to_stdout
```

Comment:
why `end` is inside of the `exception` block, not after it?

Reply:
To my understanding, when testing a method's output, the method should be contained within a do/end block or curly braces:
https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/output-matcher
With that in mind,  I'll change the do and end blocks to curly brackets to make it easier to read. Thanks, Justyna

Reply:
Sorry! I forgot about this syntax :D